### PR TITLE
build: update aspect_bazel_lib to 1.27.2, update recommended CI bazelrc settings

### PR DESCRIPTION
**Pull request by jbedard**
Primarily to pickup https://github.com/aspect-build/bazel-lib/commit/91eeca7d40e22a9f96671444f792fa3bb2545d96
See: https://github.com/aspect-build/bazel-lib/issues/381

## Test plan

CI

### CODE REVIEW

File: .aspect/bazelrc/ci.bazelrc [hunk 1]
```diff
@@ -1,40 +1,22 @@
-# Aspect recommended Bazel flags for all projects
```
Comment (jbedard) [inline .aspect/bazelrc/ci.bazelrc:1]: This file was out of date because `"ci"` was missing in the BUILD.

**Review by jhchabran — APPROVED** [review]

**Comment by jhchabran** [comment]
@cla-bot check

**Review by valerybugakov — APPROVED** [review]

**Comment by jbedard** [comment]
@valerybugakov @jhchabran I think the remaining CI failure is due to unrelated bazel errors?

**Comment by jhchabran** [comment]
@jbedard yes, it is. We haven't touched the pipeline code for these for a while and others keep pushing code breaking stuff because the buildfiles are not updated. 

We plan to fix this next week. Meanwhile, you can simply use `aspect/*` or whatever else to avoid having those issues 🙏 

I repushed your branch under a different name which is still linked to this PR because the commit is the same, so it should be all good for this one once it passes.

**Pull request merged**